jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/ObjectsFactory.java

```diff
@@ -254,4 +256,34 @@ public static Post getPostWithComments() {
         return post;
     }
 
+    public static Map<String, Object>  getNewsModelWithSpecChars(){
```

Comment:
Model? This method should create a post, a topic or something else. But there is no such entity as model.. Makes sense to use `getDefaultTopic()` or something like that where you replace the body in the test. Let's have a talk in the evening if you need further explanation.